test/physics/em/UrbanMsc.test.cc

```diff
@@ -304,5 +304,5 @@ TEST_F(UrbanMscTest, msc_scattering)
                                      -0.98020130119347};
 
     EXPECT_VEC_SOFT_EQ(expected_fstep, fstep);
-    EXPECT_VEC_SOFT_EQ(expected_angle, angle);
+    EXPECT_VEC_SOFT_NEAR(expected_angle, angle, 1e-10);
```

Comment:
```suggestion
    EXPECT_VEC_NEAR(expected_angle, angle, 1e-10);
```

Reply:
Yep... noticed that it wasn't building when vecgeom was disabled. I'm going to open another small PR to make this change, and to build (but don't run) when vecgeom is disabled, and to do a few more MSC tweaks.